library/src/main/java/com/meisolsson/githubsdk/core/DateQuery.java

```diff
@@ -0,0 +1,19 @@
+package com.meisolsson.githubsdk.core;
+
+import org.joda.time.format.ISODateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.Date;
+
+public class DateQuery {
```

Comment:
I'm not sure if having this proxy class is okay, or if we would rather the consumer just pass in a date string. With this implementation you have to pass something like `new DateQuery(new java.util.Date())`